modules/acquia_cms_article/composer.json

```diff
@@ -4,7 +4,10 @@
     "description": "Provides an Article content type and related configuration.",
     "license": "GPL-2.0-or-later",
     "require": {
-        "drupal/acquia_cms_person": "dev-1.x"
+        "drupal/acquia_cms_person": "1.x-dev"
+    },
+    "extra": {
+      "branch-alias" : {"dev-develop" : "1.x-dev"}
```

Comment:
I have one question, since we are updating dependent module version to get `1.x-dev` how we will handle this when pushing modules changes on D.O for the release?

Reply:
we will be removing above and creating the release PR with specific release version and then Tag, later revert the commit to move it back to dev.

Reply:
We have to do extra work before release of modules by replace the correct version of module. We should have alternative for this one to do extra work each time.